packages/dashboard/README.md

```diff
@@ -134,18 +170,21 @@ Once resolved, we will be able to see basic information about the DID broken dow
 3. Set the variables in .env according to how you want your hosted sidetree setup.
 4. `docker-compose build`
 5. `docker-compose up`
-6.  At this point you should be able to test sidetree locally, to go to http://localhost:8080 in your browser.
+6. At this point you should be able to test sidetree locally, to go to http://localhost:8080 in your browser.
 7. gcp cloud console setup https://github.com/transmute-industries/wikis/blob/master/docs/setup-gcp-continous-deployment.md
 8. Install gcp cloud sdk command line tools, configure it with the project and authenticate with the service account.
 9. Build and push the docker container to gcp.
+
```

Comment:
probably best to move GCP instructions to separate file.

Reply:
Updated PR moving this to /docs folder.